### PR TITLE
two seconds is better

### PR DESCRIPTION
**Pull request by shadowKiller69**

### CODE REVIEW

File: blink.ino [hunk 1]
```diff
@@ -18,7 +18,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(1000);               // wait for a second
+  delay(2000);               // wait for a second
   digitalWrite(led, LOW);    // turn the LED off by making the voltage LOW
   delay(2000);               // wait for a second
```
Comment (card4ash) [inline blink.ino]: I believe i would be better if the light was off for four seconds on for just 2

**Comment by card4ash** [comment]
if you make that change i'll be happy to merge this

**Comment by shadowKiller69** [comment]
i change it to 4 seconds

**Comment by card4ash** [comment]
#1 is good one

**Pull request merged**